calblueprint/siren
# Noah/doc setup

# ✨ New in this PR ✨

## One liner: what did you do? 🚨 

Mapped documents to doc container based on client case type(s)

## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
 - [x] Create Firebase Schema for case types and their necessary documents
 - [ ] Map documents to DocContainer based on client cases

## How can the reviewer test your code? 👩‍💻
** Make sure your client profile has a cases collection with at least one document. In the document, create an id, status, and type field. See below for an example.**

<img width="1552" alt="Screen Shot 2021-12-04 at 14 27 38" src="https://user-images.githubusercontent.com/63211322/144726422-6e0ca31b-20ba-4d41-8b21-2df85a491212.png">

Run `npm start`
Navigate to upload screen, and see case container + documents



## Any bugs you encountered or still having trouble with? 🐛
_describe how you solved them such that someone can follow your explanation 

This bug stems from having missing fields in the Case type in 'types/types.tsx'. Fix this by ensuring that the id, status, and type fields all exist in the document from your `cases` collection.
 
<img width="1077" alt="Screen Shot 2021-12-03 at 13 26 41" src="https://user-images.githubusercontent.com/63211322/144726507-9b02cad4-408d-4784-bf39-91723cc19b9a.png">



## Resources 📔
_link online resources and related PRs_ 


🧜‍♀️ cc: @shannonbonet @phoebeli23
